app/src/main/java/org/wikipedia/util/StringUtil.kt

```diff
@@ -0,0 +1,199 @@
+package org.wikipedia.util
+
+import android.text.Spanned
+import android.text.SpannedString
+import android.widget.EditText
+import android.widget.TextView
+import androidx.annotation.IntRange
+import androidx.core.text.HtmlCompat
+import com.google.gson.Gson
+import okio.ByteString.Companion.encodeUtf8
+import org.apache.commons.lang3.StringUtils
+import org.json.JSONArray
+import java.text.Collator
+import java.text.Normalizer
+
+object StringUtil {
+    private const val CSV_DELIMITER = ","
+
+    @JvmStatic
+    fun listToCsv(list: List<String?>): String {
+        return list.joinToString(CSV_DELIMITER)
+    }
+
+    @JvmStatic
+    fun csvToList(csv: String): List<String> {
+        return delimiterStringToList(csv, CSV_DELIMITER)
+    }
+
+    @JvmStatic
+    fun delimiterStringToList(delimitedString: String,
+                              delimiter: String): List<String> {
+        return delimiter.toRegex().split(delimitedString).filter { it.isNotBlank() }
```

Comment:
That's it!  But do we actually need `toRegex()` here?  Can it just be `delimitedString.split(delimiter)`?